frontend/src/theme/components/InlineMessage.ts

```diff
@@ -0,0 +1,52 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+export type InlineMessageVariant = 'info' | 'error' | 'warning'
+
+export const InlineMessage: ComponentMultiStyleConfig = {
+  parts: ['messagebox', 'item', 'icon'],
+  baseStyle: {
+    messagebox: {
+      padding: '8px',
+    },
+    item: {
+      display: 'flex',
+      py: ['1rem', '1rem', '0.5rem'],
+      px: '1rem',
```

Comment:
The padding seems to be constant at 1rem, don't think there's a difference between breakpoints